src/Components/EnvironmentPicker/EnvironmentPickerManager.ts

```diff
@@ -13,10 +13,15 @@ import {
 export const getEnvironmentDisplayText = (env: string | IAzureResource) => {
```

Comment:
![image](https://user-images.githubusercontent.com/57726991/186237532-3e84ec91-67b5-42ae-acb3-aca73163aeae.png)

Reply:
had commented and marked as "resolved" this on the original PR (basically reverted name changes)